Source/DafnyLanguageServer/Workspace/VerificationProgressReporter.cs

```diff
@@ -78,6 +78,25 @@ IDiagnosticPublisher diagnosticPublisher
     foreach (var module in document.Program.Modules()) {
       foreach (var topLevelDecl in module.TopLevelDecls) {
         if (topLevelDecl is TopLevelDeclWithMembers topLevelDeclWithMembers) {
+          if (topLevelDecl is DatatypeDecl datatypeDecl) {
```

Comment:
Does 81 need to be nested inside 80 ?

Reply:
It does not need (I'll move it as another if of the same level), but since a `DatatypeDecl` is a `TopLevelDeclWithMembers`, I thought it would be more appropriate to place it inside.